packages/date-picker/src/DatePicker/components/CalendarNav/CalendarNav.tsx

```diff
@@ -3,7 +3,7 @@ import { Icon } from "@kaizen/component-library"
 import arrowRight from "@kaizen/component-library/icons/arrow-right.icon.svg"
 import arrowLeft from "@kaizen/component-library/icons/arrow-left.icon.svg"
 import { NavbarElementProps } from "react-day-picker/types/Props"
-import styles from "../DatePicker.scss"
+import styles from "../../DatePicker.scss"
```

Comment:
WDYT about creating a new scss file for the styles that are used for this component? That's generally what we don't end up with one unruly scss file.